platform/darwin/src/MGLCircleStyleLayer.h

```diff
@@ -18,6 +18,16 @@ typedef NS_ENUM(NSUInteger, MGLCircleStyleLayerCirclePitchScale) {
 
 @interface MGLCircleStyleLayer : MGLBaseStyleLayer <MGLStyleLayer>
 
+/**
+ A predicate that corresponds to the layer's filter.
+ 
+ The predicate's left expression must be a string that identifies a feature
+ property, or one of the following special keys:
+    "$type" - identifies the feature by type.
```

Comment:
I don’t think this will look like a list in jazzy. We should either insert our own Doxygen or HTML formatting or override this documentation.